api/stories/storyNewRouter.js

```diff
@@ -62,41 +61,47 @@ router.get('/episodes/:eid', async(req, res, next) => {
       TextURL: episode[0].TextURL,
       AudioURL: episode[0].AudioURL,
       WritingPrompt: writing[0].Prompt,
-      DrawingPrompt: drawing[0].Prompt
-    }
-      res.status(200).json(data);
+      DrawingPrompt: drawing[0].Prompt,
+    };
+    res.status(200).json(data);
   } catch (error) {
     res.status(404).json({
-      message: "EpisodeNotFound",
-    })
+      message: 'EpisodeNotFound',
+    });
   }
 });
 
 //Work in progress by next cohort to add constraint to EpisodeNumber to be unique per story ID
```

Comment:
Good use of comments throughout the code.

Reply:
Thanks!